docker-compose.yml

```diff
@@ -0,0 +1,15 @@
+version: '3'
+
+services:
+
+  frontend:
+    build: ./frontend
+    ports:
+      - "3000:3000"
+    depends_on:
+      - backend
+
+  backend:
+    build: ./backend
+    ports:
+      - "8080:8080"
```

Comment:
I would suggest adding common network for frontend & backend (and later for db) to let them communicate with each other within the Docker cluster (otherwise its not possible)

    services: 
        front/backend: 
            networks:
                - my_network
        networks:
            my_network:
                driver: bridge

Reply:
Docker compose by default assigns all containers to the same network. The default `driver` is also `bridge`.
[source](https://docs.docker.com/compose/networking/)